app/frontend/src/features/communities/CommunityPage/HeaderImage.tsx

```diff
@@ -0,0 +1,44 @@
+import { Box, makeStyles } from "@material-ui/core";
+import { LngLat } from "maplibre-gl";
+import React from "react";
+
+import Map from "../../../components/Map";
+import { Community } from "../../../pb/communities_pb";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100vw",
+    height: "8rem",
+    [theme.breakpoints.up("md")]: {
+      position: "relative",
+      left: "50%",
+      right: "50%",
+      marginLeft: "-50vw",
+      marginRight: "-50vw",
+    },
+    [theme.breakpoints.up("md")]: {
+      width: "100%",
+      height: "16rem",
+    },
```

Comment:
```suggestion
    [theme.breakpoints.up("md")]: {
      position: "relative",
      left: "50%",
      right: "50%",
      marginLeft: "-50vw",
      marginRight: "-50vw",
      width: "100%",
      height: "16rem",
    },
```
These styles could be merged?

Reply:
Whoops, the bottom one is supposed to be desktop